### DISCUSSION TITLE
ViolationLS in interleaved search

### DISCUSSION THREAD

**Discussion by ozan-cb** [google/or-tools#4296] [Feature requests] [answered]
I have a scheduling problem where the violation_ls worker finds many good solutions quickly. However, to obtain deterministic results, I have to use interleaved search in my application.

I'm not sure why this worker cannot be enabled in interleaved search, but it would be great if it could be done.

**Comment by lperron** [comment, marked as answer]
it is implemented on main. It will be available in 9.11 (this month).

You can compile from source to get it.

**Comment by ozan-cb** [comment]
Good to know, thank you!
